invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/DocumentsDetails.js

```diff
@@ -77,9 +77,11 @@ export default class DocumentsDetails extends Component {
             <DocumentPanel />
           </Container>
           <Container className="document-tags">
-            <ILSParagraphPlaceholder linesNumber={1} isLoading={isLoading}>
-              <DocumentTags />
-            </ILSParagraphPlaceholder>
+            <Responsive minWidth={Responsive.onlyTablet.minWidth}>
```

Comment:
Just to confirm, so now we will have:
* < tablet: show mobile version
* > tables: show desktop version
right?

basically one breakpoint...

Reply:
Yes, I've checked and  I'm using the breakpoint which works for the layout I have proposed. But I will recheck all the possibilities again just to be sure